presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/DbResourceGroupConfigurationManagerFactory.java

```diff
@@ -50,6 +50,7 @@ public ResourceGroupConfigurationManager create(Map<String, String> config, Reso
             Bootstrap app = new Bootstrap(
                     new JsonModule(),
                     new DbResourceGroupsModule(),
+                    binder -> binder.bind(ResourceGroupConfigurationManagerContext.class).toInstance(context),
```

Comment:
The reason to use the annotation is to avoid binding the configuration context. Bind the string directly like this:
```java
binder.bind(String.class).annotatedWith(ForEnvironment.class).toInstance(context.getEnvironment())
```
Then you can remove the provides method